lib/router.js

```diff
@@ -9,6 +9,7 @@ var debug = require('debug')('koa-router');
 var compose = require('koa-compose');
 var HttpError = require('http-errors');
 var methods = require('methods');
+var mergeDescriptors = require('merge-descriptors');
```

Comment:
In general we try to have as few dependencies as possible. Adding a package for these few lines is kind of heavy.